libs/app-updater/main/src/lib/init-app-updater.ts

```diff
@@ -14,14 +14,12 @@ export const initAppUpdater = (
   appUpdaterService: AppUpdaterService | MockAppUpdaterService
 ) => {
   ipcMain.removeHandler(AppUpdaterIpcEvents.Check)
-  ipcMain.handle(AppUpdaterIpcEvents.Check, () => {
-    return appUpdaterService.check()
-  })
+  ipcMain.handle(AppUpdaterIpcEvents.Check, () => appUpdaterService.check())
```

Comment:
Nitpick, but it's always better to have brackets. It may look shorter, but it's just easier to see changes within a separate lines.

Reply:
https://github.com/mudita/mudita-center/pull/2610/commits/dfd43e915d26f27cba3eb82bc4eb1ae7281f629d